ledger/participant-integration-api/src/main/scala/platform/store/appendonlydao/JdbcLedgerDao.scala

```diff
@@ -87,46 +87,55 @@ private class JdbcLedgerDao(
   override def currentHealth(): HealthStatus = dbDispatcher.currentHealth()
 
   override def lookupLedgerId()(implicit loggingContext: LoggingContext): Future[Option[LedgerId]] =
-    dbDispatcher.executeSql(metrics.daml.index.db.getLedgerId)(storageBackend.ledgerId)
+    dbDispatcher
+      .executeSql(metrics.daml.index.db.getLedgerId)(storageBackend.ledgerIdentity)
+      .map(_.map(_.ledgerId))(DEC)
```

Comment:
minor, not necessarily needs to change: alternatively if you do the .map(_.ledgerId) in the executeSql block, then we do not need the extra mapping, neither an EC.
upto you